src/wmtk/TriMeshOperationExecutor.hpp

```diff
@@ -90,9 +90,11 @@ class TriMesh::TriMeshOperationExecutor
         const long new_eid);
 
     void connect_ears();
-    Tuple split_edge();
 
+    Tuple split_edge();
     Tuple collapse_edge();
+    Tuple split_edge_single_mesh();
```

Comment:
i think these should be called topology updates. and I'd prefer the multimesh logic be handled in the Mesh class rather than in the Executor. I think some pseudocode i wrote earlier in this reivew already assumed that.